src/cljs/ataru/virkailija/application/handlers.cljs

```diff
@@ -41,8 +42,23 @@
  (fn [db [_ _]]
    (close-application db)))
 
-(defn review-state-counts [applications]
-  (into {} (map (fn [[state values]] [state (count values)]) (group-by :state applications))))
+(defn review-state-counts
+  [applications]
+  (reduce
+    (fn [acc {:keys [application-hakukohde-reviews]}]
```

Comment:
Could this be cleaned up abit? Maybe by extracting some sub-parts with descriptive names?